packages/ui-extensions/docs/surfaces/checkout/reference/helper.docs.ts

```diff
@@ -456,6 +456,13 @@ The contents of the token are signed using your shared app secret.  The optional
         ],
       },
     },
+    storage: {
+      description: '',
+      codeblock: {
+        title: 'Storage',
+        tabs: getExtensionCodeTabs('storage'),
+      },
+    },
```

Comment:
- [ ]  @lucasuyezu made these changes
 - [ ]   @lucasuyezu approves these changes for the 2024-10 stable release.

[Storage: Add documentation examples](https://github.com/Shopify/checkout-web/commit/4442630fa51eb5ff0a4f32c7edebdcac38fcc765)